static/docs/cookie-agreements.md

```diff
@@ -0,0 +1,16 @@
+## Cookies
+&nbsp;
+
+Cookies are small files containing an identity code, which are sent to your device and stored on your browser or hard drive. These cookies allow us to distinguish you from another user and provide you with a unique browsing experience. The cookies will not cause harm to your device.
+&nbsp;
+
+Most browsers allow you to prevent (from accepting) certain cookies, to require your consent prior to storing a cookie, or block cookies altogether through your browser’s privacy preferences menu. Please find out more about controlling cookies on www.aboutcookies.org.
+&nbsp;
+
+Deltares uses two types of cookies for this website: functional cookies (for making the website work properly), and analytical cookies in anonymous mode.    
```

Comment:
gebruiken  we die? Of  later?

Reply:
I think later. 
The text in the cookie message needs to be addepted by Daniel. This is incomplete at the moment